src/Tooltip/Tooltip.js

```diff
@@ -314,7 +319,7 @@ module.exports = kind(
 	* @private
 	*/
 	positionChanged:function () {
-		Popup.prototype.positionChanged.apply(this, arguments);
+		Popup.prototype.positionChanged && Popup.prototype.positionChanged.apply(this, arguments);
```

Comment:
`positionChanged` was behaving strangely when the value was in a binding. Them _seemed_ to remedy the problem.